pom.xml

```diff
@@ -2,62 +2,23 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <packaging>pom</packaging>
+    <modules>
+        <module>project-management-application-api</module>
+        <module>project-management-application-app</module>
```

Comment:
From my point of view, I like to see the following structure for pom.xml files:

<project ...
<modelVersion>...</modelVersion>

<parent> ... </parent>

<properties>...</properties>

<modules>....</modules>

<dependencies>....</dependencies>

<build>....</build>

<profiles>...</profiles>
</project>